packages/cf-component-label/example/basic/component.js

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
-import Label from 'cf-component-label';
+import { applyTheme } from 'cf-style-container';
+import { Label as LabelUnstyled, LabelTheme } from 'cf-component-label';
+
+const Label = applyTheme(LabelUnstyled, LabelTheme);
```

Comment:
We now export styled components by default. So this should go into `./src/index.js` and you should 

```js
export { Label, LabelUnstyled, LabelTheme }
```